var/spack/repos/builtin/packages/py-lvis/package.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyLvis(PythonPackage):
+    """Python API for LVIS dataset."""
+
+    pypi = "lvis/lvis-0.5.3.tar.gz"
+
+    version("0.5.3", sha256="55aeeb84174abea2ed0d6985a8e93aa9bdbb60c61c6db130c8269a275ef61a6e")
+
+    depends_on("py-setuptools", type="build")
+    depends_on("py-cycler@0.10:", type=("build", "run"))
+    depends_on("py-cython@0.29.12:", type=("build", "run"))
+    depends_on("py-kiwisolver@1.1:", type=("build", "run"))
+    depends_on("py-matplotlib@3.1.1:", type=("build", "run"))
+    depends_on("py-numpy@1.18.2:", type=("build", "run"))
+    depends_on("opencv@4.1.0.25:+python3", type=("build", "run"))
+    depends_on("py-pyparsing@2.4.0:", type=("build", "run"))
+    depends_on("py-python-dateutil@2.8:", type=("build", "run"))
+    depends_on("py-six@1.12:", type=("build", "run"))
+    depends_on("py-pycocotools", type=("build", "run"))
```

Comment:
Where is this requirement coming from?

Reply:
```python
>>> import lvis
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/jchits/workspace/spack/opt/spack/linux-rhel7-skylake_avx512/gcc-11.2.0/py-lvis-0.5.3-6tzocas6smv6xa6kne2bejwsluie3rgw/lib/python3.10/site-packages/lvis/__init__.py", line 2, in <module>
    from lvis.lvis import LVIS
  File "/home/jchits/workspace/spack/opt/spack/linux-rhel7-skylake_avx512/gcc-11.2.0/py-lvis-0.5.3-6tzocas6smv6xa6kne2bejwsluie3rgw/lib/python3.10/site-packages/lvis/lvis.py", line 15, in <module>
    import pycocotools.mask as mask_utils
ModuleNotFoundError: No module named 'pycocotools'
```

Reply:
Thanks. Can you add a comment  about that?